style.css

```diff
@@ -114,7 +126,537 @@ p {
+  line-height: 24px;
+  letter-spacing: 3%;
+  -webkit-border-radius: 6px;
+  -moz-border-radius: 6px;
+  -ms-border-radius: 6px;
+  -o-border-radius: 6px;
+}
+
+.hidden1 button {
+  width: 133px;
+  height: 40px;
+  border-radius: 6px;
+  font-style: normal;
+  font-size: 17px;
+  color: #6070ff;
+  background-color: white;
+  border: 1px solid #7f8cff;
+  line-height: 24px;
+  letter-spacing: 3%;
+}
+
+.contact-form button {
+  height: 50px;
+  border-radius: 10px;
+  border-color: #cfd8dc;
+  border-style: none;
+  width: 40%;
+  font-size: 17px;
+  background-color: white;
+  color: #6070ff;
+}
+
+.contact-form button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.hidden button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.hidden1 button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.hidden2 button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.hidden3 button:hover {
+  background-color: #6070ff;
+  color: white;
+  border-radius: 100px;
+}
+
+.hidden33 button:hover {
+  background-color: #6070ff;
+  color: white;
+  border-radius: 100px;
+}
+
+.hidden4 button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+#stories-header {
+  font-size: 30px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  margin-top: 10px;
+  font-weight: bolder;
+  margin-bottom: 2px;
+  padding-bottom: 1px;
+  width: 55%;
+}
+
+#stories-header2 {
+  font-size: 30px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  margin-top: 10px;
+  font-weight: bolder;
+  margin-bottom: 2px;
+  padding-bottom: 1px;
+  width: 55%;
+}
+
+#stories-header1 {
+  font-size: 30px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  margin-top: 10px;
+  font-weight: bolder;
+  margin-bottom: 2px;
+  padding-bottom: 1px;
+  width: 55%;
+}
+
+.abt {
+  position: relative;
+}
+
+.about {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgb(250, 253, 255);
+  padding-top: 6em;
+  padding-left: 37px;
+  padding-right: 2px;
+  min-width: 358px;
+  left: -0.1px;
+  top: -52px;
+  border-top-right-radius: 50px;
+  background-color: white;
+  padding-bottom: 150px;
+}
+
+.about p {
+  padding-left: 0%;
+  color: #344563;
+  margin-bottom: 10px;
+  padding-bottom: 0%;
+  margin-top: 1px;
+  padding-top: 0%;
+}
+
+.about h2 {
+  font-size: 30px;
+  line-height: 40px;
+  line-height: 80%;
+  text-align: left;
+  color: #172b4d;
+  font-weight: bold;
+  padding-bottom: 1px;
+  margin-bottom: 20px;
+  padding-top: 1px;
+  margin-top: 1px;
+}
+
+.languages > ul {
+  display: flex;
+  flex-direction: column;
+  padding-left: 0%;
+  margin-left: 0%;
+}
+
+.languages1 {
+  display: none;
+}
+
+.languages li {
+  list-style: none;
+  position: relative;
+  left: -35px;
+}
+
+.frame-skill li {
+  list-style: none;
+}
+
+.languages1 ul li {
+  list-style: none;
+  position: relative;
+  left: -35px;
+}
+
+#language {
+  position: absolute;
+  left: 65%;
+  top: 455px;
+}
```

Comment:
please fix your top. your arrowll.pgn must be on the same line with the languages ​​as in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848) design